dgraph/cmd/live/run.go

```diff
@@ -618,6 +652,33 @@ func setup(opts batchMutationOptions, dc *dgo.Dgraph, conf *viper.Viper) *loader
 	return l
 }
 
+// populateNamespace fetches the schema and extracts the information about the existing namespaces.
+func (l *loader) populateNamespaces(ctx context.Context, dc *dgo.Dgraph, isGalaxyOp bool) error {
+	if !isGalaxyOp {
+		l.namespaces[opt.namespaceToLoad] = struct{}{}
```

Comment:
why do we do this? Please add a comment.
Also, why do this only for the GalaxyOp?

Reply:
Done. Thanks.